### PR TITLE
restructuring cci config

### PR DESCRIPTION
**Pull request by lhein**
Signed-off-by: Lars Heinemann <lhein.smx@gmail.com>

### CODE REVIEW

**Comment by apupier** [comment]
"cci config" --> What's the link with https://gitlab.mw.lab.eng.bos.redhat.com/dev-platform/cci-config ?

**Comment by apupier** [comment]
missign the issue number in the message

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,20 +1,18 @@
...
-  version: 2
-  build:
-    jobs:
-      - build:
-          filters:
-            branches:
-              ignore: /^dependabot.*$/
-          context: sonarcloud
+common_env: &common_env
+  DOCKER_VERSION: 17.12.1-ce
```
Comment (apupier) [inline .circleci/config.yml:4]: why do you need to pecify a DOcker version?

**Comment by lhein** [comment]
cci == circle ci

File: .circleci/config.yml (hunk 1 above)
Comment (lhein) [reply .circleci/config.yml:4]: it is not required. remember, I am experimenting.

**Pull request merged**